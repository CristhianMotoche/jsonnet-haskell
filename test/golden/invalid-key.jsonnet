{
  [null] : "null",
  [3 + "3"] : "3 + 3",
  [3 % 3]: function(x) x
}
