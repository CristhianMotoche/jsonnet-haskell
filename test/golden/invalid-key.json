Invalid key: number
    ./test/golden/invalid-key.jsonnet:4:4,4:9